gammapy/estimators/utils.py

```diff
@@ -111,7 +111,7 @@ def find_peaks(image, threshold, min_distance=1):
     return table
 
 
-def estimate_exposure_reco_energy(dataset, spectral_model=None):
+def estimate_exposure_reco_energy(dataset, spectral_model=None, normalize=True):
```

Comment:
A docstring explaning the argument `normalize` could be usefull

Reply:
Done